templates/config.hocon.tmpl

```diff
@@ -74,6 +74,13 @@ collector {
       timeLimit = ${time_limit_ms}
     }
   }
+  telemetry {
+    disable = ${disable}
```

Comment:
Collector endpoint needed for telemetry?

Reply:
It defaults to the Snowplow collector for Telemetry: https://github.com/snowplow/stream-collector/blob/53dab2dbbe644f4aa50b337ecdc7dce50b746e4b/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala#L146

It can be changed in the collector hocon, however for simplicity I think either this or disabling telemetry perhaps feels right?

Reply:
Updated to use the same url as the telemetry module